### DISCUSSION TITLE
My all deployment pages in github are marked as dangerous sites.. but they are just simple projects like calculator and my portfolio etc..

### DISCUSSION THREAD

**Discussion by SuhainaFathimaM** [community/community#133058] [Other Feature Feedback, Questions, & Ideas] [answered]
### Select Topic Area

Question

### Body

My all deployment pages in github are marked as dangerous sites.. but they are just simple projects like calculator and my portfolio etc.. How can I  make the page as normal??

**Comment by itvinl** [comment]
**Verifique dentro das configurações do GitHub Pages, veja se o domínio configurado está correto, ou se não há alguma outra configuração que possam estar causando o problema de segurança.**

_Já tentou fazer o teste acessando essas páginas por outro navegador?_

**Reply by SuhainaFathimaM** [reply]
They are accessed in other browsers.. but marked as dangerous in chrome

**Reply by itvinl** [reply]
**Acabei de fazer um teste visitando seu projeto de Portfólio e descobri que o que está gerando esse erro de segurança é seu arquivo `index.html`, ao fazer o download do arquivo o Google Chrome está detectando como um suposto vírus, creio que seja algum problema dentro desse código, provavelmente algo da parte do script**

**Comment by Damandeep-S** [comment]
Ensure your GitHub Pages are properly configured to use HTTPS. GitHub Pages automatically provides HTTPS for *.github.io domains, but make sure it's correctly set up,
Go to your repository settings on GitHub.
Scroll down to the "GitHub Pages" section.
Ensure the "Enforce HTTPS" option is checked.

**Reply by SuhainaFathimaM** [reply]
Yes i have checked.. but it is still dangerous

![Screenshot 2024-07-22 130747](https://github.com/user-attachments/assets/84887131-fa6a-4452-99ee-a049ff365f74)

**Comment by Damandeep-S** [comment]
Use tools like Google Search Console or SUCURI SiteCheck to scan your site for any security issues and resolve them accordingly. And moreover you should check the console and network tabs in developer tools for any errors or warnings.

**Reply by SuhainaFathimaM** [reply]
okay

**Comment by knarrff** [comment]
It is hard to say what cause this, as the mechanism isn't open. Try to not refer to domains you don't own, e.g., try adding real contact info, not something like fake phone numbers and an email address which does not resolve. Also, in my case I got a dialog to report this site as "not suspicious".

**Comment by aishamc** [comment]
Go to the GitHub Pages settings for your project. Make sure the domain and other settings are correct. Something might be set up incorrectly.

Try Different Browsers: See if the problem happens in other browsers. Sometimes, security warnings are only in one browser.

Look at Your Code: Check your index.html and any scripts or links you’re using. Sometimes, security warnings come from certain code or links that look suspicious.

Scan Your Site: Use tools like Google Search Console or SUCURI SiteCheck to scan your site for any issues. These tools can help find problems that might be causing the warnings.

Check for Errors: Open the developer tools in your browser (press F12) and look at the Console and Network tabs. This can show you any errors or issues that might be causing the warnings.

Use Real Contact Info: Make sure you’re not using fake contact details on your site. Sometimes, fake info can trigger warnings.

**Comment by davevad93** [comment, marked as answer]
Hi @SuhainaFathimaM ,

This usually happens with repos hosted on GitHub Pages cloning the design of popular websites like `Facebook`, `Amazon`, `Instagram`, `Twitter`, `Netflix`, `Spotify`. That's a `Google` security policy, for more information see the Google Safe Browsing documentation on [Social engineering (phishing and deceptive sites)](https://developers.google.com/search/docs/advanced/security/social-engineering?hl=en). However I've also noticed it happens with other repos, for example here #132906 , the issue could be related to the Google Safe Browsing phishing list that is flagging all your projects hosted on GitHub Pages. Take a look at this [comment](https://github.com/orgs/community/discussions/30261#discussioncomment-3467665) from discussion #30261 and check if it helps you to solve it.
